consult.el

```diff
@@ -562,8 +562,21 @@ This command obeys narrowing."
                       :history 'consult-line-history
                       :lookup (lambda (candidates x) (cdr (assoc x candidates)))
                       :default (car candidates)
+		      :initial initial
                       :preview (and consult-preview-line #'consult--preview-line))))))
 
+;;;###autoload
+(defun consult-line-symbol-at-point ()
+  "Search for a symbol at point."
+  (interactive)
+  (consult-line (thing-at-point 'symbol)))
+
+;;;###autoload
+(defun consult-line-from-isearch ()
+  "Search by lines from isearch string."
+  (interactive)
+  (consult-line isearch-string))
```

Comment:
swiper seems to have something like this: 

~~~ elisp
(if isearch-regexp
                   isearch-string
                 (regexp-quote isearch-string))))
~~~

Add something like that too?

Reply:
Sure.